fraqbot/Local/shakespeare.py

```diff
@@ -1,49 +1,59 @@
-from Legobot.Lego import Lego
 import logging
 import os
 import random
 import sys
 
+from Legobot.Lego import Lego
+
+
 logger = logging.getLogger(__name__)
+LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
 
+if LOCAL_DIR not in sys.path:
```

Comment:
This block was actually not necessary for what you submitted. I kept it though because I imported the helpers module below, and the local dir needs to be in the path to be able to find helpers.